### DISCUSSION TITLE
zu HMT2250 keine Verbindung

### DISCUSSION THREAD

**Discussion by onweb65** [tbnobody/OpenDTU#1323] [Q&A]
Ich habe hier mitlerweile 4 HMT-2250 im Einsatz und einen Bausatz von 
https://shop.blinkyparts.com/de/OpenDTU-CMT-Deine-Auswertung-fuer-deine-Balkonsolaranlage-kompatibel-zu-Hoymiles-HMS-und-HMT-Serie-CMT-Modul/blink238342.2 zur Kommunikation erworben.

...gekauft, vorgelötet, den Rest habe ich fertig gelötet und gebaut.. meine Lötererfahrung sind sehr gut.

Aber keine Verbindung möglich:
Status: 
CMT2300A Status | konfiguriert
CMT2300A Chip-Status | verbunden

Aber, ich sehe nicht einen Einzigen meiner HMT-2250
Der originale Router von Hoymile verbindet sich problemlos, also grundsätzlich funktioniert es

Macht es Sinn an der CMT2300A Frequenz zu spielen? Hatte aber selbst in 1 MHz-Schritten nichts gefunden

Protokoll ungefähr so:
14:05:58.524 > All missing
14:05:58.524 > Nothing received, resend whole request
14:05:58.524 > TX SystemConfigPara 865.00 MHz --> 15 90 11 83 97 80 13 37 44 80 05 00 64 FD B1 1F 00 00 00 00 00 00 00 00 06 3D E9 
14:05:58.818 > RX Period End

jemand eine Idee? Module defekt?

**Comment by tbnobody** [comment]
> Der originale Router von Hoymile verbindet sich problemlos, also grundsätzlich funktioniert es

Du meinst DTU? Ein Doppelbetrieb ist nicht möglich. Original DTU ausschalten, mindestens 15min warten (dann fällt der Inverter auf seine default Frequenz zurück) und nochmal testen.

**Comment by onweb65** [comment]
Was wäre die default Frequenz, weil den "Einzelbetrieb" hatte ich schon zuvor getestet, da andere Beiträge auch darauf eingingen.

Nun ist der Original DTU seit 20 min Spannungslos, aber bei meinem open DTU blinkt/leuchtet weder die 2. LED, noch hat sich der Verbindungsstatus (All missing) geändert. 

15:22:28.978 > All missing
15:22:28.978 > Nothing received, resend whole request
15:22:28.978 > TX SystemConfigPara 868.00 MHz --> 15 90 60 01 63 80 13 37 44 80 05 00 64 FD C3 0D 00 00 00 00 00 00 00 00 47 4A B8 
15:22:29.237 > RX Period End

**Comment by huste511** [comment]
Standardfrequenz mit 865 Mhz funktionert.
Was für eine Antenne dran? Die Sendeleistung mal erhöhrt?
Bis 14dBm sind bei Dipol in D erlaubt, mal in kleinen Schritten rantasten.
Wenn eine Verbindung da ist, kannst Du (leider nur in der Konsole) die Signalstärke der einzelnen MWR in dBm sehen.

**Comment by onweb65** [comment]
Ich sehe nun 2 von 4, ..14dBm eingestellt und der openDTU ist nur ca. 10 m von den HMT entfernt (hinter Fenster mit direkter Blickrichtung auf PV)
Antenne sieht man hier: 
https://shop.blinkyparts.com/de/OpenDTU-CMT-Deine-Auswertung-fuer-deine-Balkonsolaranlage-kompatibel-zu-Hoymiles-HMS-und-HMT-Serie-CMT-Modul/blink238342.2 
Diese ist im Prinzip nicht anders als die am Original, welches deutlich stabiler Daten empfing (leider nur alle 15 min und in China-Cloud)

**Reply by huste511** [reply]
Mit wieviel dBm kommen die zwei Geräte rein?
10m ist nun wirklich keine weite Stecke, da so sollten 5dBm ausreichen.
Die Polarisierung der Antennen macht auch noch *etwas* aus. Bei den MWR liegen die meist horizontal.
Die Antenne an der Basis soltle daher auch am besten liegen und mit möglichst viel Seitenfläche den MWR präsentieren.

PS: bei den Antennen gibt es nicht nur innen und außgengewinde sondern unabhängig davon auch noch weibchen und männchen.
Nicht das Du nun Dose auf Dose verschraubt hast?

**Comment by lasse79** [comment]
Ich hänge mich mal mit dran... selbes Problem  wobei bei mir der WR mal am anfang kurz zu sehen war, nach ca 30min war er wer und nicht mehr zu finden. Abstand zwischen WR und openDTU ca 3m ...

**Comment by Timwaehner** [comment]
Gibt es zu diesem Problem etwas neues? Habe ehrlich gesagt kaum Ahnung von OpenDTU und der ganzen Software dahinter. Deshalb habe ich auch die vorgeflashte Variante bei Blinkyparts bestellt. Ordentlich verlötet aber exakt das gleiche Problem, wie oben beschrieben. Abstand zum WR beträgt ca 2m

**Comment by dhjupi** [comment]
Auch bei mir, ich sehe nicht einen Einzigen meiner 3 HMT-2250
und habe auch einen Blinkyparts Vorgelötet  am laufen.
Timwaehner hast du es schon geschaft?

**Comment by dhjupi** [comment]
Jetzt wird auch bei mir alles angezeigt, nachdem ich die DTU Pro ca 15 min Getrennt habe.

**Comment by mbnoone** [comment]
Für mich war wichtig: Nach Einstellungen|DTU-Einstellungen|Seriennummer gehört keinesfalls die Seriennummer des Wechselrichters!
(ist mir passiert, v24.1.21, @tbnobody: könnte die Firmware entdecken und warnen)

**Reply by 1212tom** [reply]
Gibt es hier schon Neuigkeiten? Habe das Problem auch ... der HMS 1600 wird angezeigt nur die HMT 2250 nicht

**Reply by tutti83** [reply]
Hallo, hast du das Problem lösen können??

**Comment by 1212tom** [comment]
Gibt es schon eine Lösung für dieses Problem

**Comment by 1212tom** [comment]
Also ich habe auf diese Antennen gewechselt und habe jetzt seit 14 Tagen kein Problem mehr 

DollaTek 5Pcs U.FL IPEX zu... https://www.amazon.de/dp/B07QXPN3YR?ref=ppx_pop_mob_ap_share

**Comment by mighty1977** [comment]
Hallo zusammen,
ich habe das gleiche Problem bei mir.
Ich betreibe 1 HMT-2250-6T an der OpenDTU in Kombination mit dem Smart Micro Solar.
Wenn ich den WR einmal von den MPPT's trenne, also spannungslos mache und dann neu verbinde/starte, dann verbindet er sich wieder und arbeitet eine ganze weile problemlos.
Irgendwann dann gibt die Verbindung auf und kommt auch nicht wieder zustande, bis zum nächsten "Neustart"!
Das heisst für mich, das die Funkverbindung grundsätzlich funktioniert, jedoch irgendwann keine Antwort mehr vom WR zurück kommt!?
Sieht für mich eher nach einem Problem mit dem WR aus!?


11:13:25.652 > Fetch inverter: 1382xxxxxxxx
11:13:25.652 > Request SystemConfigPara
11:13:25.652 > Queue size - NRF: 0 CMT: 6
11:13:25.824 > RX Period End
11:13:25.824 > All missing
11:13:25.824 > Nothing received, resend whole request
11:13:25.824 > TX PowerControl 865.00 MHz --> 51 90 78 09 82 80 18 49 72 81 00 00 B0 01 A1

**Reply by SW-Niko** [reply]
Das Problem haben schon mehrere Leute reported. 
Aber ich bin mir nicht sicher, ob das ein Problem von Hoymiles ist, oder ob wir den Inverter in diesen Zustand bringen.

Hast du probiert, ob du den Inverter über den Menüpunkt "zurücksetzen" auf der WebUI wieder zum laufen bringst?
Wie schaut deine Funkstatistik aus?

**Reply by mighty1977** [reply]
Danke für deine Antwort,

ich habe es mehrfach probiert den WR via Web-UI neu zu starten.
Es sieht so aus als wenn er einfach jeden Befehl ignoriert!
Könnte es sein, das der Hersteller einfach vermeiden möchte, das häufige Leistungsänderungen gemacht werden!?
Ist ja immerhin im regulären Betrieb, mit Boardmitteln von Hoymiles, nicht möglich, automatische Leistungsänderungen zu machen!
Mir scheint es daher fast so als wenn es einen Zähler gibt in der WR Firmware, der dies unterbindet!?
Wenn man es genau nimmt, würde ein Microwechselrichter im regulären Betrieb, also direkt am Panel, immer Nachts zurück gesetzt werden.
An einer Batterie ist es theoretisch möglich, das er 24/7 läuft!

Ich habe daher momentan einen Workarround, der leider nicht jedem möglich sein wird.
Ich betreibe den WR an 3 Zendure SolarFlow Hub 2000, welche derzeit via Zeitplan, 4 mal am Tag die Ausgabeleistung für 2 Minuten auf 0 W setzt.
Dadurch werden die WR zurückgesetzt und die Funkverbindung wieder aktiviert.

Wem das nicht möglich sein sollte, könnte das evtl. mit Zeitschaltuhr und Relais vor den MPPT's realisieren.

Schön wäre es natürlich, wenn man wüsste, ob ein Reset per Befehl den gleichen Effekt hätte, dann könnte man diesen Automatismus möglicherweise direkt in der OpenDTU integrieren!?

**Reply by SW-Niko** [reply]
Ich bin jetzt etwas verwirrt.
Das kann auch daran liegen, das ich OpenDTU-OnBattery verwende und nicht OpenDTU.

>Ich habe es mehrfach probiert den WR via Web-UI neu zu starten.

<img width="350" height="160" alt="grafik" src="https://github.com/user-attachments/assets/6d6b797c-e66f-4ad1-b2c0-1ee7aefb12d2" />

Wir reden über den Schalter "Neustarten", richtig?

>Könnte es sein, das der Hersteller einfach vermeiden möchte, das häufige Leistungsänderungen gemacht werden!?

Das ich möglich, aber ich halte es für unwahrscheinlich. An meinem System mache ich im Durchschnitt  ca. alle 15 Sekunden (tags) eine Leistungsänderung und hatte bis jetzt noch keinen Ausfall.

>Schön wäre es natürlich, wenn man wüsste, ob ein Reset per Befehl den gleichen Effekt hätte, dann könnte man diesen Automatismus möglicherweise direkt in der OpenDTU integrieren!?

Bei OpenDTU-OnBattery gibt es diese Möglichkeit. Vielleicht gibt es das bei OpenDTU auch?

<img width="600" height="65" alt="grafik" src="https://github.com/user-attachments/assets/f36c5e7f-2a81-4342-9568-e8ab2bd5d482" />

**Reply by mighty1977** [reply]
> Ich bin jetzt etwas verwirrt. Das kann auch daran liegen, das ich OpenDTU-OnBattery verwende und nicht OpenDTU.
> 
> > Ich habe es mehrfach probiert den WR via Web-UI neu zu starten.
> 
> <img alt="grafik" width="350" height="160" src="https://private-user-images.githubusercontent.com/129541740/573892108-6d6b797c-e66f-4ad1-b2c0-1ee7aefb12d2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzUzODMwNTcsIm5iZiI6MTc3NTM4Mjc1NywicGF0aCI6Ii8xMjk1NDE3NDAvNTczODkyMTA4LTZkNmI3OTdjLWU2NmYtNGFkMS1iMmMwLTFlZTdhZWZiMTJkMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNDA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDQwNVQwOTUyMzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNzViZmQ5MDRmZTg2ZjJjN2Y5ZGZiOGQ4MDA4ZWIxZGNmYTMwYTBiZDE0N2RjYTFiOWExYTZjY2Q1ZWYyYzAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5F49r99-OG939KZV7FW9JilZIj7YnHcVAV9gZYlDvPE">
> Wir reden über den Schalter "Neustarten", richtig?

Ja, genau dieser.

> 
> > Könnte es sein, das der Hersteller einfach vermeiden möchte, das häufige Leistungsänderungen gemacht werden!?
> 
> Das ich möglich, aber ich halte es für unwahrscheinlich. An meinem System mache ich im Durchschnitt ca. alle 15 Sekunden (tags) eine Leistungsänderung und hatte bis jetzt noch keinen Ausfall.

Ich hatte bis jetzt noch keine Zeit und Lust, mir mal die technischen Unterschiede zur "OnBattery" Version anzuschauen, klingt jedoch definitiv nach Unterschieden!

> 
> > Schön wäre es natürlich, wenn man wüsste, ob ein Reset per Befehl den gleichen Effekt hätte, dann könnte man diesen Automatismus möglicherweise direkt in der OpenDTU integrieren!?
> 
> Bei OpenDTU-OnBattery gibt es diese Möglichkeit. Vielleicht gibt es das bei OpenDTU auch?
> 
> <img alt="grafik" width="600" height="65" src="https://private-user-images.githubusercontent.com/129541740/573892766-f36c5e7f-2a81-4342-9568-e8ab2bd5d482.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzUzODMwNTcsIm5iZiI6MTc3NTM4Mjc1NywicGF0aCI6Ii8xMjk1NDE3NDAvNTczODkyNzY2LWYzNmM1ZTdmLTJhODEtNDM0Mi05NTY4LWU4YWIyYmQ1ZDQ4Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNDA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDQwNVQwOTUyMzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YmM3MWQ0MjIzNzMxNjJjYWNlMzZiNDE0NjBjMmNmMTA5NDJhNDYyYzkyNDU1MzQ3NjJhM2IxM2E1ZDJkNGE5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PC6OtXFPhX7D5_swJ39gFItp9SU1LhqED37eX0JlVl4">

Diese Einstellung habe ich in meiner OpenDTU Version tatsächlich nicht!

Das einfachste wäre an dieser Stelle, einmal zu checken, ob meine OpenDTU-Hardware mit der OnBattery Version zum laufen zu bringen ist!?

Parallel arbeite ich an einer Regelung, welche die Ausgabeleistung des Zendure SolarFlow Hub 2000 steuert, somit wäre die OpenDTU dann für meinen Aufbau tatsächlich hinfällig!

**Reply by SW-Niko** [reply]
>Parallel arbeite ich an einer Regelung, welche die Ausgabeleistung des Zendure SolarFlow Hub 2000 steuert..

Schau dir OpenDTU-OnBattery mal genauer hinsichtlich der Unterstützung von Smart-Batteries an. Dazu zählt auch Zendure SolarFlow Hub 2000. 
Eine Regelung mit verschiedenen Konfigurationsmöglichkeiten gibt es.
